frontend/test/metabase/scenarios/question/filter.cy.spec.js

```diff
@@ -648,6 +648,38 @@ describe("scenarios > question > filter", () => {
     popover().contains(/case/i);
   });
 
+  it("should enable highlighting suggestions with keyboard up and down arrows (metabase#16210)", () => {
+    openReviewsTable({ mode: "notebook" });
+    cy.findByText("Filter").click();
+    cy.findByText("Custom Expression").click();
+
+    cy.get("[contenteditable='true']")
+      .click()
+      .type("c");
+
+    // First highlighted selection is Created At
+    // "C" is wrapped in a <span>
+    // so we must look for string "reated At"
+    cy.findByText("reated At")
```

Comment:
This is a perfect example of where `cy.contains()` really shines and is better to use than `cy.findByText()`

`cy.contains("Created At")` works and is exactly what you need here.

Reply:
Thanks!